src/color_laws.jl

```diff
@@ -137,3 +137,39 @@ function cal00_invum(x::Real, Rv::Real)
     return 1.0 + k / Rv
 
 end
+
+
+"""
+    gcc09(λ::Real, Rv=3.1)
+    gcc09(λ::Quantity, Rv=3.1)
+
+Gordon, Cartledge, & Clayton (2009) dust law.
+
+If `λ` is a `Unitful.Quantity` it will be automatically converted to Å and the
+returned value will be `UnitfulAstro.mag`.
+"""
+function gcc09(λ::Real, Rv = 3.1)
+    x = aa_to_invum(λ)
+    return gcc09_invum(x, Rv)
+end
+
+gcc09(λ::Quantity, Rv::Real = 3.1) = gcc09(ustrip(u"angstrom", λ), Rv) * u"mag"
+
+function gcc09_invum(x::Real, Rv::Real)
+
+    a = zeros(typeof(Rv))
+    b = zeros(typeof(Rv))
+    if 3.3 <= x && x <= 11.0  # NUV
+        a = 1.894 - 0.373 * x - 0.0101 / ((x - 4.57)^2 + 0.0384)
+        b = -3.490 + 2.057 * x + 0.706 / ((x - 4.59)^2 + 0.169)
+    else
+        return 0.0x
+    end
+    if 5.9 <= x && x <= 11.0  # far-NUV
```

Comment:
```suggestion
    if 5.9 <= x <= 11.0  # far-NUV
```

Reply:
Will keep this in mind from next time! :smiley: